src/TaskItem.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const TaskItem = ({ task, toggleCompleted }) => {
+    const {id, name, due, completed} = task;
+
+    const toggleCompletedClick = () => {
+        toggleCompleted(id);
+    }
+    return (
+        <tr className={(new Date(due)) <= (new Date()) ? 'overdue' : 'notDueYet'}>
```

Comment:
**Readability**
Not required but you can make the comparison here a variable
```js
const isOverdue = new Date(due)) <= (new Date()
...
        <tr className={isOverdue ? 'overdue' : 'notDueYet'}>
```
Isn't it so much cleaner and easier to read?